### ISSUE TITLE
Applications section

### ISSUE THREAD

**Issue by sofroniewn** [napari/napari-tutorials#43]
Following #42 we are adding an `Applications` section that can contain more advance tutorials highlighting specific applications.

As discussed with @tlambert03 it's probably good to have these separate from the more basic introductory tutorials so that new comers and beginners aren't overwhelmed.

I also see these as slightly different from blog posts as I'd like them to be maintained and updated with time (as following a napari breaking api change), whereas a blog post I see as more capturing the state at a moment in time (though obviously they can be updated too).

I'd also love for these to be community contributed as much as possible too, so maybe after we have a couple more example ones in we could make a `Contributing Guide` to set expectations. I see these as more didactic and in-depth then just here's cool data viewed in napari, for that I'd like to have some form of community contributed gallery, as discussed with @jni 

How does this sound to everyone?

To also keep the convo going here are a couple ideas for posts:

- [x] lazy loading and compute with dask - see #42
- [ ] lazy parameter sweep with dask - see [gist](https://gist.github.com/d-v-b/5dbdb7513d07e3360cbe5c0f3d5cacb6)
- [ ] custom mouse functions + 1D plots - see https://github.com/napari/napari/pull/675#issuecomment-553672258
- [ ] napari + binder remote desktop - see [repo](https://github.com/manics/jupyter-omeroanalysis-desktop/tree/napari-binder)
- [ ] napari + omero - see [tweet](https://twitter.com/will_j_moore/status/1187705800518705154)
- [ ] painting labels + custom segmentation code - see [repo](https://github.com/transformify-plugins/segmentify)

**Comment by ruhmamehek** [comment]
Hi, I find 'lazy parameter sweep with dask' interesting, and I would like to try writing a tutorial on this. Can I please take this up?

**Comment by jni** [comment]
@ruhmamehek yes, please give it a go! I would recommend having a look at the example in my [blog post](https://ilovesymposia.com/2019/10/24/introducing-napari-a-fast-n-dimensional-image-viewer-in-python), which was a real-world use case mixing both an image and a points layer.

**Comment by sofroniewn** [comment]
That would be amazing! It might be great to use [magicgui](https://magicgui.readthedocs.io/en/latest/examples/napari_parameter_sweep/) made by @tlambert03 (who is also a napari core developer) to provide GUI interactivity for the parameter sweep

**Comment by tlambert03** [comment]
... but, erm, let me update that magicgui tutorial a bit before you get too far with it 😅.  there is at least one thing (`register_type`) in there that is a bit out of date.

**Comment by tlambert03** [comment]
updated the param sweep example in the docs

**Comment by ruhmamehek** [comment]
Hi, I have added a tutorial for lazy parameter sweep with dask. I have added an example for parameter sweep using magicgui with napari as well. Looking forward to your reviews and suggestions.

**Comment by tlambert03** [comment]
Thank you @ruhmamehek!  I'm excited to look through it and will comment shortly!
